developer-guide/index.md

```diff
@@ -120,9 +128,23 @@ In order for the **first distribution channel** to be created, you’ll be promp
 
 We’ll be gathering feedback from our custom-built _Pepperoni-app_, hence we decide to name this channel the _Pepperoni channel_.
 
-### Copying the JSON configuration from the form editor
+### Retrieving the JSON configuration
+
+We can now retrieve the JSON configuration of the form and channel we just created. In order to retrieve the configuration, we head to the *Channel setup* page where we have two options:
+
+1. We copy the JSON directly from this page. This way suites best scenarios where internet connection is limited.
+2. We copy the URL to retrieve the JSON programmatically. This method is best to automatically keep your form up to date with its latest published version.
+
+Both options lead to the same value as the JSON we can copy is the same as the one we can retrieve from the URL. Also, the URL is predictable since it is always composed as:
+
+```
+https://delivery.eu.getfbk.com/channels/<CHANNEL_ID>/<LOCALE>
+```
+
+where:
 
-We can now copy the JSON configuration of the form and channel we just created from the channel setup screen.
+- `CHANNEL_ID` is the id of the channel associated to the form
+- `LOCALE` is the language code specifying the localization of the form. This is either a specific locale or the values `main`, an alias for the main language. Because this alias will always exists for every configuration, it may be used as a fallback in case a specific localized configuration is not found.
```

Comment:
```suggestion
- `LOCALE` is the language code specifying the localization of the form. This is either a specific locale or the value `main`, alias for "main language". Because this alias will always exists for every configuration, it may be used as a fallback in case a specific localized configuration is not found.
```

Reply:
done